plugins/modules/aws_glue_job.py

```diff
@@ -188,7 +207,11 @@
 '''
 
 from ansible_collections.amazon.aws.plugins.module_utils.aws.core import AnsibleAWSModule
-from ansible_collections.amazon.aws.plugins.module_utils.ec2 import camel_dict_to_snake_dict
+from ansible_collections.amazon.aws.plugins.module_utils.ec2 import (camel_dict_to_snake_dict,
```

Comment:
nit:
```
from ansible.module_utils.common.dict_transformations import camel_dict_to_snake_dict
```
This function was migrated over into 'common' because it was useful.  While there are a lot of instances of this being imported from module_utils.ec2 we should start moving them over.